src/orders/components/OrderAddTransaction/OrderAddTransaction.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { order } from "@saleor/orders/fixtures";
+import Wrapper from "@test/wrapper";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+
+import OrderAddTransaction from "./OrderAddTransaction";
+
+describe("OrderAddTransaction", () => {
+  it("renders skeleton when order is loading", () => {
+    render(
+      <Wrapper>
```

Comment:
**issue**: Using Wrapper makes this test non-isolated which defies the purpose of unit test. Values required by contexts should be mocked (example for reference: `ExitFormDialog.test.tsx`). This happens in multiple tests in this PR.

Reply:
I would argue the opposite: our components have to be used within a context provider, so mocking import of the hook instead of creating a fake context provider gives us less confidence in our tests.

In the `Wrapper` we have context providers for all needed use cases:
- fake apollo provider
- intl provider with only 1 locale: EN
- locale provider with locale set to EN
- date provider, so that date doesn't change between test runs
- timezone provider, same as above
- theme provider - without this, our components simply won't render, because they expect a theme from macaw

Reply:
That's no longer a unit test, but rather an integration/functional. You should break down your transaction component into smaller pieces and test them separately by unit tests. Instead, you are wrapping the entire functionality, providing many contexts just for testing your functionality as one piece.

It's ok, you can keep that because you put so much effort into this, but... think about it: you don't use everything from your wrapper, you use only specifying things, you rely only on certain fields from given context, and so on, and you should test your components against them first (interface segregation), and if that works - then integration/functional test can add some value.

Whatever breaks now either in your code or these wrappers, it will be difficult to find where the bug comes from your code, or that wrapper - as you tightly couple it